### ISSUE TITLE
Separate windows from `Rebound` project into individual projects

### ISSUE THREAD

**Issue by Lamparter** [IviriusCommunity/Rebound#68]

**Comment by IviriusMain** [comment]
Does this mean the UAC prompt is gonna be in a separate project and all that?

**Comment by Lamparter** [comment]
Yes, unless you want it to be part of the cpl project which would make sense. I want the CPL project to also be a collection of miscellaneous settings; OR we could organise a `Rebound.Control.*` namespace inside a Visual Studio folder so it sort-of appears as one project and the solution explorer isn't too cluttered and neither is the repo.

**Comment by IviriusMain** [comment]
The second option should do the trick

**Comment by IviriusMain** [comment]
The installer will be removed so this issue is redundant.

**Comment by Lamparter** [comment]
No
